static/app/components/performance/vitalsAlert/constants.tsx

```diff
@@ -0,0 +1,15 @@
+import {VitalsKey} from './types';
+
+export const INDUSTRY_STANDARDS: Record<VitalsKey, number> = {
```

Comment:
Would be nice if this had some comment about where these numbers came from

Reply:
@evanpurkhiser Yea let me do that in a follow-up